delfin/drivers/huawei/oceanstor/rest_client.py

```diff
@@ -244,4 +244,11 @@ def get_all_pools(self):
 
     def clear_alert(self, sequence_number):
         url = "/alarm/currentalarm?sequence=%s" % sequence_number
-        return self.call(url, method="DELETE", log_filter_flag=True)
+
+        # Result always contains error code and description
+        result = self.call(url, method="DELETE", log_filter_flag=True)
+        if result['error']['code'] != 0:
```

Comment:
if result['error']['code']:   ==> this is enough. No need to check with !=0.

Reply:
Updated